specifyweb/frontend/js_src/lib/components/Toolbar/QueryTablesEdit.tsx

```diff
@@ -37,7 +37,13 @@ export function QueryTablesEdit({
     />
   );
 }
-
+// TODO: temp fix, remove this, use to hide geo tables for COG until 9.8 release
+const GEO_TABLES = new Set([
+  'CollectionObjectType',
+  'CollectionObjectGroup',
+  'CollectionObjectGroupJoin',
+  'CollectionObjectGroupType',
+]);
```

Comment:
Make this the same with variable defined in Tables.tsx? @CarolineDenis or @melton-jason

Reply:
@CarolineDenis I don't see this resolved. This should be a 1 minute change...?

Reply:
Fixed in https://github.com/specify/specify7/pull/5032/commits/76dceab023068707a7ca43d3fa5f86c00510b6a3